docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,50 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
```

Comment:
Even so syntactically correct, it does not "flow" so well imo. I also don't like that we have to stand-alone sentences to start the overview. I suggest to start more like this:

"A profile is an instance of the Minishift VM along with all of its configuration and state. The profile feature allows you to create and manage these isolated instances of Minishift."

@robin-owen, @LalatenduMohanty - WDYT?